### PR TITLE
Adding week1 to the main

### PR DESCRIPTION
**Pull request by CamilleGayat**

### CODE REVIEW

**Review by Pierre2103 — DISMISSED** [review]

File: documents/management/weekly_reports/week_1.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Next week planning
+
+The next week we will focus on the Functional specifications mainly.
+
+Next week concern's:
+- Finish entirely the Functional specifications
+- Finish the Project charter
+- Finish the Mock-ups
+- Gantt
```
Comment (MistzSoftware) [inline documents/management/weekly_reports/week_1.md]: You wrote "Finish" for others line but not form Gantt and RACI what will you do with these 2?

File: documents/management/weekly_reports/week_1.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- RACII
```
Comment (MistzSoftware) [inline documents/management/weekly_reports/week_1.md]: RACI not RACII

**Review by MistzSoftware — CHANGES_REQUESTED** [review]

**Comment by Pierre2103** [comment]
line 6 - Use only dashes(-) or only fewer symbols(<) to allow consistency
line 8 - " The task didn't started yet " --> replace " didn't " by " haven't "
line 8 - " The task is at these beginnings and only the basis are made or are being made " --> " The ongoing task is still in progress and well started "
line 8 - " The task is progressing well and the team enter in the heart of it " --> " entering in the heart of it " OR " The task is progressing well and the task is coming to life. "
line 8 - " The end of the task approach and the team must continue at a good pace " --> " The task is coming to an end, however few changes has to be done. "
line 8 - " This is the last part of the task, finish the task and pass it in review. " --> " the final phase of the task has been completed and is currently being reviewed by the QA "
line 8 - " The task is entirely done and included " --> " The task is entirely completed and integrated in the repository "

**Comment by Pierre2103** [comment]
line 22 - " Github Repository creation " --> " Creation of the Github Repository "
line 26 - " A meeting was made " --> " A meeting was done "
line 27 - " Another one was made " --> " Another one was done "
line 27 - " 10 minutes before the end " --> " 10 minutes before the end of the day "
line 28 - " Two else were made " --> " Two more meetings were done "
line 28 - " at the beginning and the end of the project  " --> " at the start of the day and at the end of the day "
line 28 - " to share the progress and motivate the team "  --> " to maintain progression of the project "

**Comment by Pierre2103** [comment]
line 32 - replace this:

> " This week the project advanced a lot, The decision to all make a different mock-up and then sharing them together to keep the best ideas of everyone was made, then with these ideas, 2 groups were made to prepare two mock-ups to present to the client. But we need to be careful about the deadline for the Functional specifications. "

by this :

> " The kick-off of the project was fairly satisfying, we decided as a team to come up with different mock-ups which we will present to the client.  Two main mock-ups has been keeped, and will be sent next week to the client."

line 34 - replace this:

> " Globally, all the members of the team worked a lot and brought of their own in the advancement. They are motivated and feels great, some are confident with flutter and dart. "

by this :

> " Globally, all team members has contributing to allow the team to come up with theses final mock-ups, the team members motivation is on-point and few team members feels confident with Dart and Flutter technologies. "

line 38 to line 45 - replace everything by 

> " The next weeks main concern will be focus on the Functional Specifications, and in parallel completing the Project Charter, Wait for the response of the client, finishing the GANTT Charter and finishing the RACI Matrix "

**Review by MistzSoftware — DISMISSED** [review]

File: documents/management/project_charter.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+| <img src=https://ca.slack-edge.com/T019N8PRR7W-U05T1QG3AKE-faa57959e1b3-192 width="150" height="150"> | GAGNEPAIN Mathias | Technical Leader | [GitHub](https://github.com/MathiasGagnepain) \| [LinkedIn](https://www.linkedin.com/in/mathias-gagnepain-426a131b0/) |
+| <img src=https://ca.slack-edge.com/T019N8PRR7W-U05SWDH3MLM-32416239ba9d-192 width="150" height="150"> | DESPAUX Guillaume | Technical Writer | [GitHub](https://github.com/GuillaumeDespaux) \| [LinkedIn](https://www.linkedin.com/in/guillaume-despaux-084b10206/) |
+| <img src=https://ca.slack-edge.com/T019N8PRR7W-U05SJQXQHM5-1b3487dfe573-192 width="150" height="150"> | LAURENT Ian | Software Engineer | [GitHub](https://github.com/Ianlaur) \| [LinkedIn](https://www.linkedin.com/in/ian-h-laurent/) |
+| <img src=https://ca.slack-edge.com/T019N8PRR7W-U02FHCYEJJD-3f3110d47354-192 width="150" height="150"> | GORIN Pierre | Quality Assurance | [GitHub](https://github.com/Pierre2103) \| [LinkedIn](https://www.linkedin.com/in/pierre-gorin-61a784221/) |
+
+## Stakeholders
+
+| **Name** | **Role** | **Represented by** | **Contact** |
```
Comment (MistzSoftware) [inline documents/management/project_charter.md]: don't forget to remove contact column

**Review by MistzSoftware — CHANGES_REQUESTED** [review]

**Comment by MistzSoftware** [comment]
@Pierre2103 @CamilleGayat don't forget to complete the review of the pull request.
If not completed I will close the PR tomorrow at the end of the day.

**Comment by Pierre2103** [comment]
Perfect !

**Review by Pierre2103 — APPROVED** [review]

**Review by Pierre2103 — APPROVED** [review]

**Review by MistzSoftware — APPROVED** [review]

**Pull request merged**